com.github.IsmaelMartinez.teams_for_linux.desktop

```
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Name=Microsoft Teams for Linux
+Comment=Unofficial client for Microsoft Teams for Linux
+Exec=teams-for-linux %U
+Terminal=false
+Type=Application
+Icon=teams-for-linux.png
```

```suggestion
Icon=com.github.IsmaelMartinez.teams_for_linux
```
The icon name, desktop file, appdata file should have the same name as your application id.